llama_stack/providers/inline/post_training/torchtune/utils.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+# Copyright (c) Meta Platforms, IAny, nc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+from typing import Any, Callable, Dict
+
+import torch
+from llama_models.sku_list import resolve_model
+
+from torchtune.models.llama3 import llama3_tokenizer, lora_llama3_8b
+from torchtune.models.llama3._tokenizer import Llama3Tokenizer
+from torchtune.models.llama3_2 import lora_llama3_2_3b
+
+LORA_MODEL_TYPES: Dict[str, Any] = {
```

Comment:
1. can we put all the related things are necessary for a model and club them together into a single data structure which captures all this information? 
2. can you say a bit more about what "checkpoint model type" represents and how torchtune uses that constant?